usm docs fixes (#24092)
DataDog/documentation: content/en/universal_service_monitoring/setup.md

{{% /tab %}}

{{< /tabs >}}
{{< /collapse-content >}}

{{< collapse-content title="Non-IIS service discovery on Windows" level="h4" >}}
<div class="alert alert-info">

> Universal Service Monitoring is available in <strong>beta</strong> to monitor non IIS services running on a Windows machines.

Reviewer: ```suggestion
Universal Service Monitoring is available in <strong>beta</strong> to monitor non IIS services running on Windows environments.
```

</div>

Requires Agent version 7.42 or greater.

{{< tabs >}}
{{% tab "Configuration file" %}}

Add the following configuration to the `system-probe.yaml`:

```yaml
service_monitoring_config:
  enabled: true

>   process_service_inference:

Reviewer: ```suggestion
process_service_inference:
```
I this supposed to be indented ?

Author: yes, it is nested on intention

>     enabled: true

Reviewer: ```suggestion
  enabled: true
```

Author: same as above
